src/io/camera.js

```diff
@@ -78,26 +84,34 @@ module.exports = function Camera(options) {
 
   function success(stream) {
     var video;
+    window.localStream = stream;
+    isOnCam = stream;
+    isCamera = true;
     if (webRtcOptions.context === "webrtc") {
-      video = webRtcOptions.videoEl;
+      video = document.getElementById("webCamVideoEl");
```

Comment:
Hi Forcha, this is looking really good. I wanted to note that these lines where we access an options parameter instead of directly using a string are intended to make it easy to point the entire system at a different HTML setup without having to change multiple spots within the code. So if it's possible to preserve these in all the changes you made that would be great. I'm happy to help explain how this gets set and how it works if that's helpful!

Reply:
Thank you @jywarren. I have made the requested changes on line 91
```diff

+  video = webRtcOptions.videoEl;
-  video = document.getElementById("webCamVideoEl");

```